trace/HTTP.md

```diff
@@ -0,0 +1,25 @@
+# Tracing HTTP
+
+## Client
+
+* Client span name format
```

Comment:
in zipkin we use the http method name due to fixed cardinality. For platforms that support route templates, that's the better choice. https://github.com/openzipkin/zipkin/issues/1874